core/src/main/java/com/avast/metrics/core/multi/SummaryMonitor.java

```diff
@@ -37,61 +37,61 @@
  *     Monitor summaryMonitor = monitor.named("path");
  *     Monitor instanceMonitor = summaryMonitor.named("instance");
  *
- *     Monitor monitor = MultiMonitor.of(instanceMonitor, summaryMonitor);
+ *     Monitor monitor = SummaryMonitor.of(instanceMonitor, summaryMonitor);
  *
  *     this.errors = monitor.newMeter("errors");
  *     this.requests = monitor.newMeter("requests");
  * </code></pre>
  * <p>
  * <pre><code>
  *     Monitor summaryMonitor = monitor.named("path");
- *     Monitor monitor = MultiMonitor.of(summaryMonitor, summaryMonitor);
+ *     Monitor monitor = SummaryMonitor.of(summaryMonitor, summaryMonitor);
  *     ConcurrentMap&lt;String, Meter&gt; requests = new ConcurrentHashMap&lt;&gt;()
  *
  *     // Dynamically create a new meter if not present in the cache and increment its value
  *     requests.computeIfAbsent(topic, t -&gt; monitor.named(t).newMeter("requests")).mark();
  * </code></pre>
  */
-public class MultiMonitor implements Monitor {
+public class SummaryMonitor implements Monitor {
```

Comment:
This breaks backward compatibility. Please extend `MultiMonitor` from `SummaryMonitor` and make it `@deprecated`, don't forget factory methods. It may be removed in version 3.x or so.

Reply:
It's VERY sad to break backward compatibility two weeks after releasing new major version, especially when it waited for the release 3 months :disappointed:   
Plese, folow Michals solution.

Reply:
Done